kolibri/plugins/coach/assets/src/views/reports/ReportsLessonResourceLearnerListPage.vue

```diff
@@ -82,12 +114,23 @@
 
 <script>
 
+  import { LastPages } from '../../constants/lastPagesConstants';
   import commonCoach from '../common';
+  import ReportsResourceLearners from './ReportsResourceLearners';
+  import ReportsResourcesStats from './ReportsResourcesStats';
 
   export default {
     name: 'ReportsLessonResourceLearnerListPage',
-    components: {},
+    components: {
+      ReportsResourceLearners,
+      ReportsResourcesStats,
+    },
     mixins: [commonCoach],
+    data() {
+      return {
+        viewByGroups: Boolean(this.$route.query && this.$route.query.groups),
```

Comment:
I think `this.$route.query` is always going to be truthy, since it's an empty object if no query parameters are provided.